pillar/components/upgrade.sls

```diff
@@ -17,13 +17,20 @@
 
 upgrade:
   sw_list:
-    - motr
-    - s3server
-    - hare
-    - ha.cortx-ha
-    - sspl
-    - uds
-    - csm
+    - motr:
+      sls: components.motr
```

Comment:
I would prefer going more granular here, like `components.motr.upgrade`
Also, would it be possible to accept list of sls?
```
upgrade:
  sw_list:
    - motr:
      sls:
        - components.motr.backup
        - components.motr.upgrade
```
This would help shut down a particular flow path during Debug sessions.

Reply:
It's a base_sls path to a component states and not the list of states to apply. Actually It's not about an upgrade. Makes sense to move to another place keeping here only list of sw to consider during the upgrade. I will work on that more. Thanks

Reply:
moved general sw specs to commons pillar, upgrade.sls now includes only the list of sw we plan to upgrade